packages/plugin-hardhat/README.md

```diff
@@ -1,39 +1,24 @@
-# OpenZeppelin Buidler Upgrades
+# OpenZeppelin Hardhat Upgrades
 
-[![Docs](https://img.shields.io/badge/docs-%F0%9F%93%84-blue)](https://docs.openzeppelin.com/upgrades-plugins/buidler-upgrades)
-[![NPM Package](https://img.shields.io/npm/v/@openzeppelin/buidler-upgrades.svg)](https://www.npmjs.org/package/@openzeppelin/buidler-upgrades)
+[![Docs](https://img.shields.io/badge/docs-%F0%9F%93%84-blue)](https://docs.openzeppelin.com/upgrades-plugins/hardhat-upgrades)
+[![NPM Package](https://img.shields.io/npm/v/@openzeppelin/hardhat-upgrades.svg)](https://www.npmjs.org/package/@openzeppelin/hardhat-upgrades)
 
-**Buidler plugin for deploying and managing upgradeable contracts.** This package adds functions to your Buidler scripts so you can deploy and upgrade proxies for your contracts. Depends on `ethers.js`.
+**Hardhat plugin for deploying and managing upgradeable contracts.** This package adds functions to your Hardhat scripts so you can deploy and upgrade proxies for your contracts. Depends on `ethers.js`.
 
 ## Installation
 
 ```
-npm install --save-dev @openzeppelin/buidler-upgrades
-npm install --save-dev @nomiclabs/buidler-ethers ethers # peer dependencies
-```
-
-And register the plugins in your [`buidler.config.js`](https://buidler.dev/config/):
-
-```js
-usePlugin('@nomiclabs/buidler-ethers');
-usePlugin('@openzeppelin/buidler-upgrades');
-```
```

Comment:
What are your thoughts on this section and why did you remove it?

(The TypeScript part that follows this is indeed no longer necessary.)

Reply:
I removed it by mistake, adding it back now.